<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardOverview</class>
 <widget class="QDialog" name="CardOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Overvew</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="maximumSize">
      <size>
       <width>491</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Suit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Point</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Subtype</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Package</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>221</width>
       <height>521</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="cardLabel">
          <property name="maximumSize">
           <size>
            <width>217</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="cardDescriptionBox">
          <property name="maximumSize">
           <size>
            <width>217</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="malePlayButton">
            <property name="maximumSize">
             <size>
              <width>105</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Play (male)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="femalePlayButton">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>104</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Play (female)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="playAudioEffectButton">
          <property name="text">
           <string>Play audio effect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getCardButton">
          <property name="maximumSize">
           <size>
            <width>217</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Get this card</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Card Overview</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
